inference-engine/tests/functional/plugin/myriad/common/myriad_common_test_utils.hpp

```diff
@@ -5,12 +5,12 @@
 #pragma once
 
 #include <cstdlib>
-#include <iostream>
+#include <string>
 
 namespace CommonTestUtils {
 namespace vpu {
 
-bool CheckMyriad2() {
+inline bool CheckMyriad2() {
```

Comment:
Why do you do that? :) If it's a way to avoid ODR violation, so please move definition into source file.

Reply:
Done